src/parser/lessParser.ts

```diff
@@ -146,7 +146,7 @@ export class LESSParser extends cssParser.Parser {
 	}
 
 	public _parseDetachedRuleSetBody(): nodes.Node {
-		return this._tryParseKeyframeSelector() || super._parseRuleSetDeclaration();
+		return this._tryParseKeyframeSelector() || super._parseRuleSetDeclaration() || super._parseRuleset();
```

Comment:
Should be:
`this._tryParseKeyframeSelector() || this._tryParseRuleset(true)  || super._parseRuleSetDeclaration()`
Rule sets start with selectors that also contains ':'. _tryParseRuleset will try to parse this a selector, but will reset the scanner if it turns out to be something else.